lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -70,7 +70,13 @@ def routing_spec?(args)
         end
 
```

Comment:
Specs are good, but do you mind to add some documentation on how it works as comments for this class?

Reply:
Sure thing. I've updated the documentation and have re-pushed this branch.